index.md

```diff
@@ -362,7 +362,9 @@ Here are some examples of messages and how the parameters would be represented a
 
       :dan!d@localhost PRIVMSG #chan Hey!   ->  ["#chan", "Hey!"]
 
-As the last two examples show, a trailing parameter (a parameter prefixed with `':'`) is another regular parameter. Once the `':'` is stripped, software MUST just treat it as another param.
+      :dan!d@localhost PRIVMSG #chan ::-)   ->  ["#chan", ":-)"]
+
+As these examples show, a trailing parameter (a final parameter with a preceding `':'`) has the same semantics as any other parameter, and MUST NOT be treated specially once the `':'` is stripped.
```

Comment:
```suggestion
      :dan!d@localhost PRIVMSG #chan ::-)   ->  ["#chan", ":-)"]

As these examples show, a trailing parameter (a final parameter with a preceding `':'`) has the same semantics as any other parameter, and MUST NOT be treated specially or stored separately once the `':'` is stripped.
```

Minor copyediting, and clarification in terms of 'stored separately' on the final bit of the explanation.